exporters/otlp/otlp_integration_test.go

```diff
@@ -426,3 +428,85 @@ func TestNewExporter_withHeaders(t *testing.T) {
 	require.Len(t, headers.Get("header1"), 1)
 	assert.Equal(t, "value1", headers.Get("header1")[0])
 }
+
+func TestNewExporter_withMultipleAttributeTypes(t *testing.T) {
+	mc := runMockCol(t)
+
+	defer func() {
+		_ = mc.stop()
+	}()
+
+	<-time.After(5 * time.Millisecond)
+
+	exp, _ := otlp.NewExporter(
+		otlp.WithInsecure(),
+		otlp.WithReconnectionPeriod(50*time.Millisecond),
+		otlp.WithAddress(mc.address),
+	)
+
+	defer func() {
+		_ = exp.Stop()
+	}()
+
+	tp, err := sdktrace.NewProvider(
+		sdktrace.WithConfig(sdktrace.Config{DefaultSampler: sdktrace.AlwaysSample()}),
+		sdktrace.WithBatcher(exp, // add following two options to ensure flush
+			sdktrace.WithBatchTimeout(15),
```

Comment:
Can we attach a unit here?

```suggestion
			sdktrace.WithBatchTimeout(15 * time.Second),
```

Reply:
Hi~ we only test one span here. `15s ` maybe too long?

Reply:
Gotcha, that was more a suggestion of a unit. Please choose an appropriate one.

Reply:
done. thanks.